samples/golf/CMakeLists.txt

```diff
@@ -199,8 +189,12 @@ target_link_libraries(${PROJECT_NAME}
   ${SQLite3_LIBRARIES}  
   ${OPENGL_LIBRARIES})
 
+if (TARGET crogine)
+ target_link_libraries(${PROJECT_NAME} crogine)
```

Comment:
strictly speaking there should be an `else` condition on this to add the ${CROGINE_LIBRARIES} because that's what would be defined if we're using a pre-installed version, but there aren't any ill side-effects from adding that when it's empty so have left it for simplicity